pkg/pcr/flow.go

```diff
@@ -204,6 +208,12 @@ func (f Flow) MeasurementIDs() MeasurementIDs {
 			MeasurementIDBIOSDirectoryLevel1,
 			MeasurementIDBIOSDirectoryLevel2Header,
 			MeasurementIDBIOSDirectoryLevel2,
+			MeasurementIDPMUFirmwareInstructions,
+			MeasurementIDPMUFirmwareData,
+			MeasurementIDMicrocodePatch,
```

Comment:
In next PR let's rename it to `MeasurementIDAMDMicrocodePatch`, because Intel also has Microcodes.